extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -389,6 +390,7 @@ angular.module('oppia').component('svgFilenameEditor', {
                   width: dimensions.width + 'px'
                 };
                 $scope.$applyAsync();
+                ctrl.isLoadingIndicatorShown = false;
               };
               img.src = getTrustedResourceUrlForSvgFileName(data.filename);
             }, function(parsedResponse) {
```

Comment:
You also need to add `ctrl.isLoadingIndicatorShown = false;` when there's an error, otherwise the loading sign will keep on spinning.

Reply:
Done